jobs/flanneld/templates/config/etcd-client.crt.erb

```diff
@@ -1,2 +1,2 @@
-<%= link('etcd').p('tls.etcdctl.certificate') %>
+<%= link('etcd').p('tls.etcdctl-flanneld.certificate') %>
```

Comment:
Let's kill the link here and just let it be consumed directly. The link doesn't really give us anything here - to be entirely honest, its just led us to re-use certs in a way we shouldn't be in the first place.

Reply:
cool